src/commands/app/deploy.js

```diff
@@ -142,13 +142,10 @@ class Deploy extends BuildCommand {
 
     try {
       await runInProcess(config.hooks['pre-app-deploy'], config)
-      if (flags['feature-event-hooks']) {
-        this.log('feature-event-hooks is enabled, running pre-deploy-event-reg hook')
-        const hookResults = await this.config.runHook('pre-deploy-event-reg', { appConfig: config, force: flags['force-events'] })
-        if (hookResults?.failures?.length > 0) {
-          // output should be "Error : <plugin-name> : <error-message>\n" for each failure
-          this.error(hookResults.failures.map(f => `${f.plugin.name} : ${f.error.message}`).join('\nError: '), { exit: 1 })
-        }
+      const hookResults = await this.config.runHook('pre-deploy-event-reg', { appConfig: config, force: flags['force-events'] })
```

Comment:
Can we use constants instead of hardcoded string? `"pre-deploy-event-reg"` and  `"post-deploy-event-reg"`

Reply:
done